py/torch_tensorrt/dynamo/conversion/impl/topk.py

```diff
@@ -134,3 +134,38 @@ def sort(
         return topk_layer.get_output(0), topk_layer.get_output(1)
     else:
         return topk_layer.get_output(0)
+
+
+def topk(
```

Comment:
Hi can you consolidate the topk() and the sort() function, something like the sort() function calls the topk function with the k arg as the dim?

Also in the below the sorted argument should also be handled to return the elements in sorted order.